src/model/Column.ts

```diff
@@ -141,15 +154,22 @@ export default class Column extends AEventDispatcher {
    * @private
    */
   private compressed = false;
-  renderername: string;
-  protected rendererList: {type: string, label: string}[];
+
+
+  protected rendererInfo: IRendererInfo;
+
 
   constructor(id: string, public desc: IColumnDesc) {
     super();
     this.uid = fixCSS(id);
     this.cssClass = (<any>this.desc).cssClass || '';
-    this.renderername = (<any>this.desc).type || 'heatmapcustom';
-    this.rendererList = [];
+    this.desc.rendererType = this.desc.type;
```

Comment:
what about: `this.renderInfo.rendererType = this.desc.rendererType || this.desc.type`